.teamcity/_self/projects/WebApp.kt

```diff
@@ -330,6 +340,12 @@ object RunAllUnitTests : BuildType({
 	}
 
 	features {
+		feature {
+			type = "xml-report-plugin"
```

Comment:
This will only capture the code errors and display them in a tab. To fail the build we need `failOnMetricChange` block (like https://github.com/Automattic/wp-calypso/blob/bc3906b58e5b67a91437b5113ee6183f41700b1a/.teamcity/_self/projects/WebApp.kt#L718).

I think we should merge this to trunk first and let it spread to other builds for a couple of days. If not, the first build with any `tsc` error will fail because the metric has changed from 0 to N (I think, not 100% sure)